src/helm/benchmark/scenarios/legal_summarization_scenario.py

```diff
@@ -0,0 +1,194 @@
+    3. EurLexSum (https://arxiv.org/abs/2210.13448)
+
+    Task prompt structure
+
+        Summarize the given document.
+        Document: {tok_1 ... tok_n}
+        Summary: {tok_1 ... tok_m}
+
+    Example from MultiLexSum dataset (Short to Tiny)
+
+        Document: {This case is about an apprenticeship test that had a disparate impact
+                    on Black apprenticeship applicants. The Equal Employment Opportunity
+                    Commission (EEOC) filed this lawsuit on December 27, 2004, in U.S.
+                    District Court for the Southern District of Ohio. Filing on behalf
+                    of thirteen Black individuals and a class of similarly situated Black
+                    apprenticeship test takers, the EEOC alleged that the individuals’
+                    employer, the Ford Motor Company, as well as their union, the United
+                    Automobile, Aerospace, and Agricultural implement workers of America
+                    (the “UAW”), and the Ford-UAW Joint Apprenticeship Committee, violated
+                    Title VII of the Civil Rights Act, 42 U.S.C. § 1981, and Michigan state
+                    anti-discrimination law. The EEOC sought injunctive relief and damages
+                    for the Black apprenticeship applicants. The individuals also brought a
+                    separate class action against Ford and the UAW, and the cases were
+                    consolidated. In June 2005, both cases were resolved via a class
+                    settlement agreement. Ford agreed to pay $8.55 million and to implement
+                    a new selection process for its apprenticeship programs, and the court
+                    ordered Ford to cover attorneys’ fees and expenses. This case is closed.}
+        Summary: {2005 class action settlement resulted in Ford paying $8.55m to redesign
+                    its selection process for apprenticeship programs to address the
+                    previous process’s disparate impact on Black applicants.}
+    """
+
+    name = "summarization"
+    description = "Scenario for legal summarization tasks"
+    tags = ["summarization"]
+
+    # Mapping from HELM splits to HF splits
+    splits_mapping = {
+        TRAIN_SPLIT: datasets.Split.TRAIN,
+        VALID_SPLIT: datasets.Split.VALIDATION,  # BillSum does not have any validation split
+        TEST_SPLIT: datasets.Split.TEST,
+    }
+
+    def __init__(
+        self,
+        dataset_name: str,
+        sampling_min_length: Optional[int] = None,
+        sampling_max_length: Optional[int] = None,
+        doc_max_length: Optional[int] = None,
+    ):
+        """
+        Initializes summarization scenario.
+        Args:
+            dataset_name: String identifier for dataset. Currently
+                          supported options ["BillSum", "MultiLexSum", "EurLexSum"].
+            sampling_min_length: Int indicating minimum length (num whitespace-separated tokens) for training
```

Comment:
Likewise, min_num_tokens etc.?

Reply:
This is also copied from the summarization scenario